client/src/featureform/serving.py

```diff
@@ -630,6 +653,9 @@ def _register_model(
         self.db.insert("models", name, type)
         self.db.insert(look_up_table, name, association_name, association_variant)
 
+    def get_source_as_df(self, name, variant):
+        return self.get_input_df(name, variant)
```

Comment:
Curious if this works with the caching @aolfat added

Reply:
It does because I'm calling into the same local method that's used when serving. I've tested this manually and checked `.featureform/cache`.

Reply:
nice!